### PR TITLE
redirect reference to ssm-parameters template

### PR DESCRIPTION
**Pull request by zaro0508**
The ssm-parameters template was moved to the shared template repo
(Sage-Bionetworks/aws-infra repo).  Instead of having it here just reference
the one in the shared repo instead.

depends on Sage-Bionetworks/aws-infra#263

### CODE REVIEW

File: config/develop/synapse-login-scipooldev-params.yaml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 template_path: ssm-parameters.j2
```
Comment (boyleconnor) [inline config/develop/synapse-login-scipooldev-params.yaml]: Shouldn't this be `remote/ssm-parameters.j2` ?

File: config/prod/synapse-login-scipoolprod-params.yaml [hunk 2]
```diff
@@ -1,5 +1,8 @@
 template_path: ssm-parameters.j2
```
Comment (boyleconnor) [inline config/prod/synapse-login-scipoolprod-params.yaml]: Shouldn't this be `remote/ssm-parameters.j2` ?

**Review by boyleconnor — CHANGES_REQUESTED** [review]

File: config/develop/synapse-login-scipooldev-params.yaml (hunk 1 above)
Comment (zaro0508) [reply config/develop/synapse-login-scipooldev-params.yaml]: good catch!

Comment (zaro0508) [reply config/develop/synapse-login-scipooldev-params.yaml]: fixed

File: config/prod/synapse-login-scipoolprod-params.yaml (hunk 2 above)
Comment (zaro0508) [reply config/prod/synapse-login-scipoolprod-params.yaml]: fixed

**Review by boyleconnor — APPROVED** [review]

**Pull request merged**